pkg/gatewayserver/observability.go

```diff
@@ -202,7 +202,7 @@ var gsMetrics = &messageMetrics{
 			Name:      "downlink_tx_failed_total",
 			Help:      "Total number of unsuccessfully emitted downlinks",
 		},
-		[]string{protocol},
+		[]string{protocol, "result"},
```

Comment:
Nit: constant like `host` or `protocol`.

Reply:
Technically we use those constants for repeated labels.